include/pmacc/mappings/kernel/StrideIntervalMapping.hpp

```diff
@@ -0,0 +1,221 @@
+/* Copyright 2021 Sergei Bastrakov
```

Comment:
I believe it is in a separate commit anyways. But all in all, it does not matter or does it?

Reply:
It is needed for this PR and the functionality of this PR is so far the only client. It could have been done before in theory (but then without any client code), but it wasn't. So i would suggest keeping it as part of this PR.

Reply:
Okay